lib/elements/atom/assets.js

```diff
@@ -1,12 +1,31 @@
 const fs = require('fs');
 const path = require('path');
 
-const logo = String(fs.readFileSync(path.resolve(__dirname, '..', '..', '..', 'assets', 'logo.svg')));
+let LogoPath = '../../../assets/logo.svg';
+let LogoSmallPath = '../../../assets/logo-small.svg';
+let ScreenshotPath = '../../../assets/plotscreenshot.png';
+let DemoVideoPath = '../../../assets/demo.mp4';
+let InstallLessPath = '../../../styles/install.less';
 
-const logoSmall = String(fs.readFileSync(path.resolve(__dirname, '..', '..', '..', 'assets', 'logo-small.svg')));
+// Environment variable set by consumer repository using Webpack Plugins via config.
+if (process.env.USING_WEBPACK_FILELOADER_TO_BUNDLE) {
+  const WEBPACK_FILELOADER_PATH_KEY = "default";
 
-const screenshot = path.resolve(__dirname, '..', '..', '..', 'assets', 'plotscreenshot.png');
+  LogoPath = require('../../../assets/logo.svg')[WEBPACK_FILELOADER_PATH_KEY];
```

Comment:
I think you can probably do simpler:

```
import LogoPath from '../../../assets/logo.svg'
```

since `file-loader` should expose a standard ES6 module (which is why `['default']` also works).

Reply:
I ran into an error about mixing import and require syntax, which I hypothesized was an issue when webpack bundles into main.js, so I stuck with the require syntax.

Reply:
oh gotcha, makes sense!